src/Duets.Cli/Text/Career.fs

```diff
@@ -76,11 +76,16 @@ let shiftDurationDescription schedule =
     match schedule with
     | JobSchedule.Free shiftDuration ->
         $"""{shiftDuration} {Generic.simplePluralOf "day moment" shiftDuration} per shift"""
+    | JobSchedule.Fixed (_, shiftDuration) ->
+        $"""{shiftDuration} {Generic.simplePluralOf "day moment" shiftDuration} per shift"""
 
 let scheduleDescription schedule =
     match schedule with
     | JobSchedule.Free _ ->
         $"""No schedule, {shiftDurationDescription schedule}"""
+    | JobSchedule.Fixed (workDays, _) ->
+        let dayNames = workDays |> List.map (fun day -> day.ToString()) |> String.concat ", "
```

Comment:
Use the already built-in util to comma separate the days. Also, do not use `ToString` on the day name, we don't want to show a localized name for the day. Instead, add a new translation function in the calendar namespace and use English names.